petclinic-frontend/src/pages/Users/AllUsers.css

```diff
@@ -0,0 +1,97 @@
+
```

Comment:
I think you should've used similar css styling as the customer's get all page to keep a similar style throughout the entire website. I'm not requesting you to change it, however I think it should be something to consider when creating a new page.

Reply:
Perfect il keep that in mind